src/server.js

```diff
@@ -35,10 +37,22 @@ export default function start(
 
 	const baseRoutes = getRoutes(renderRequestMap);
 	const finalRoutes = [...routes, ...baseRoutes];
+	let tls = null;
+	if (
+		appConfig.app_server.protocol === 'https' &&
+		appConfig.app_server.key_file &&
+		appConfig.app_server.crt_file
+	) {
+		tls = {
+			key: fs.readFileSync(appConfig.app_server.key_file),
```

Comment:
I'm reading in these files because they have newlines (that need to be preserved), and that makes them super hard to add to JSON.